schema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE region (
+  id uuid PRIMARY KEY,
```

Comment:
get_random_uuid() produces very fragmented inserts so I'm going to use uuid_generate_v1mc() instead.